packages/KLabel/KLabel.vue

```diff
@@ -1,12 +1,27 @@
 <template functional>
   <label class="k-input-label">
-    <slot />
+    <KoolTip
+      v-if="props.help"
+      :label="props.help">
+      <slot />
+      <KIcon
+        icon="help"
+        height="16"
+        width="16" />
+    </Kooltip>
+    <slot v-else />
   </label>
 </template>
 
 <script>
 export default {
-  name: 'KLabel'
+  name: 'KLabel',
+  props: {
+    help: {
```

Comment:
Can we also add the ability to specify an `info` prop that uses the info icon?

Reply:
Great idea. I've added this